libs/dh/admin/data-access-api/src/lib/dh-admin-user-roles-management-data-access-api.store.ts

```diff
@@ -99,6 +110,31 @@ export class DhAdminUserRolesManagementDataAccessApiStore extends ComponentStore
     )
   );
 
+  readonly createUserRole = this.effect(
+    (userRoleCreateDto: Observable<UserRoleCreate>) => {
+      return userRoleCreateDto.pipe(
+        tap(() => {
+          this.setLoading(LoadingState.LOADING);
+          this.setCreateState(LoadingState.INIT);
+        }),
+        switchMap((userRole) =>
+          this.saveUserRole(userRole.createRole).pipe(
+            tapResponse(
+              () => {
+                this.setLoading(LoadingState.LOADED);
+                userRole.onSaveCompletedFn();
```

Comment:
Question: Shouldnt setCreateState  be called with LOADED or CREATED as well?

Reply:
Yes it should, fixed.